tests/main/snap-repair/task.yaml

```diff
@@ -3,6 +3,14 @@ summary: Ensure that snap-repair is available
 # snap-repair is not shipped on non-ubuntu
```

Comment:
maybe time to move this to tests/core  given it's core only ?

Reply:
where would we check that snap-repair is disabled on classic? Do we still need to spread test that, perhaps unit test coverage is good enough?

Reply:
mmh,  I see,   we could also split the tests in a classic one about not running (no-snap-repairs) and a core one with the rest. But if we are trying to improve the testing we should probably also  check the reverse of:

systemctl list-timers | MATCH snapd.snap-repair.timer

for classic, same for the service itself.  We don't check for that ATM afaik.

We should also double check if the conditions in the .in files for  look right, they have a TODO:UC20 atm still that is not valid anymore at least:

data/systemd/snapd.snap-repair.service.in  data/systemd/snapd.snap-repair.timer

Reply:
yeah those TODO's are wrong in the service files, I will delete them here, and also split up the test as described